### PR TITLE
Feature/39445

### PR DESCRIPTION
**Pull request by szafran89**

### CODE REVIEW

File: Magento_Customer/templates/form/register.phtml [hunk 1]
```diff
@@ -66,209 +88,100 @@
...
+                        autocomplete="off"
+                    >
+                </div>
+                <div class="password-strength">
+                    <div
+                        id="password-strength-meter-container"
+                        data-role="password-strength-meter"
+                        aria-live="polite"
+                    >
+                        <div id="password-strength-meter"
```
Comment (Aekal) [inline Magento_Customer/templates/form/register.phtml]: Move `id` attribute to new line

File: Magento_Customer/templates/form/register.phtml [hunk 2]
```diff
@@ -66,209 +88,100 @@
...
             </legend>
-            <div class="input required">
-                <label for="email_address"
-                       class="input__label"
-                >
+            <div class="divider registration__divider">
+                <span class="divider__left"></span>
+                <span class="divider__right"></span>
+            </div>
+            <div class="input registration__input  required">
```
Comment (Aekal) [inline Magento_Customer/templates/form/register.phtml]: Remove extra space before 'required'

File: Magento_Customer/templates/form/register.phtml (hunk 1 above)
Comment (szafran89) [reply Magento_Customer/templates/form/register.phtml]: moved

File: Magento_Customer/templates/form/register.phtml (hunk 2 above)
Comment (szafran89) [reply Magento_Customer/templates/form/register.phtml]: fixed

**Pull request merged**